rick-morty-app-client/package.json

```diff
@@ -40,13 +40,15 @@
     ]
   },
   "devDependencies": {
+    "@craco/craco": "^7.1.0",
     "@testing-library/jest-dom": "^6.6.3",
     "@testing-library/react": "^16.0.1",
     "@testing-library/user-event": "^14.5.2",
     "@types/jest": "^29.5.14",
     "@types/node": "^22.10.1",
     "@types/react": "^18.3.13",
     "@types/react-dom": "^18.3.1",
-    "typescript": "^4.9.5"
```

Comment:
why you change the version of typescript? with version 4.9.5 it work but with that version 5.72 not.